MODULE.bazel

```diff
@@ -3,16 +3,59 @@ module(
     version = "1.37.0-dev",
 )
 
+# All bazel_dep entries are organized alphabetically for improved readability.
+# Runtime dependencies from Bazel Central Registry
+bazel_dep(name = "apple_support", version = "1.23.1", repo_name = "build_bazel_apple_support")
 bazel_dep(name = "aspect_bazel_lib", version = "2.21.2")
+bazel_dep(name = "aspect_rules_js", version = "2.5.0")
+bazel_dep(name = "bazel_features", version = "1.38.0")
+bazel_dep(name = "bazel_skylib", version = "1.8.2")
+bazel_dep(name = "boringssl", version = "0.20250514.0")
+bazel_dep(name = "brotli", version = "1.1.0", repo_name = "org_brotli")
+bazel_dep(name = "cel-spec", version = "0.24.0", repo_name = "dev_cel")
+bazel_dep(name = "emsdk", version = "4.0.13")
 bazel_dep(name = "envoy_api", version = "1.37.0-dev")
 bazel_dep(name = "envoy_build_config", version = "1.37.0-dev")
 bazel_dep(name = "envoy_mobile", version = "1.37.0-dev")
+bazel_dep(name = "fast_float", version = "8.0.2")
+bazel_dep(name = "flatbuffers", version = "25.2.10", repo_name = "com_github_google_flatbuffers")
+bazel_dep(name = "fmt", version = "12.1.0", repo_name = "com_github_fmtlib_fmt")
+bazel_dep(name = "gazelle", version = "0.47.0", repo_name = "bazel_gazelle")
+bazel_dep(name = "googleapis", version = "0.0.0-20241220-5e258e33.bcr.1", repo_name = "com_google_googleapis")
+bazel_dep(name = "googleurl", repo_name = "com_googlesource_googleurl")
 bazel_dep(name = "gperftools", version = "2.17.2")
+bazel_dep(name = "highway", version = "1.2.0")
+bazel_dep(name = "nlohmann_json", version = "3.12.0", repo_name = "com_github_nlohmann_json")
+bazel_dep(name = "platforms", version = "1.0.0")
+bazel_dep(name = "protobuf", version = "30.0", repo_name = "com_google_protobuf")
+bazel_dep(name = "protoc-gen-validate", version = "1.2.1.bcr.1", repo_name = "com_envoyproxy_protoc_gen_validate")
+bazel_dep(name = "re2", version = "2024-07-02.bcr.1", repo_name = "com_googlesource_code_re2")
+bazel_dep(name = "rules_buf", version = "0.5.1")
+bazel_dep(name = "rules_cc", version = "0.2.13")
+bazel_dep(name = "rules_foreign_cc", version = "0.15.0")
+bazel_dep(name = "rules_go", version = "0.57.0", repo_name = "io_bazel_rules_go")
+bazel_dep(name = "rules_java", version = "8.14.0")
+bazel_dep(name = "rules_license", version = "1.0.0")
+bazel_dep(name = "rules_nodejs", version = "6.5.0")
```

Comment:
Is this required for `emsdk`? Don't see a reference to `nodejs` in current code base beyond `emsdk` and comments